src/features/Treege/Forms/JsonNodeForm.tsx

```
@@ -0,0 +1,76 @@
+      id="json-node-form"
+      className="flex flex-col h-full pb-4 min-h-0"
+      onChange={handleSubmit}
+      onSubmit={(e) => {
+        e.preventDefault();
+        e.stopPropagation();
+      }}
+    >
+      <div className="flex flex-col gap-6 h-full">
+        <Field
+          name="label"
+          children={(field) => (
```

Biome (2.1.2)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)